add account command in cli

AlverLyu opened the pull request:
1. add command of account management
2. modify help message
3. change wallet file format

tanZiWen commented:
approve

AlverLyu committed:
add account command in cli

1. add command of account management
2. modify help message
3. change wallet file format

laizy commented on account/client.go:
```diff
@@ -209,38 +231,52 @@ func (cl *ClientImpl) GetAccountByAddress(address common.Address) *Account {
-		fmt.Println("error: load wallet info failed")
-		return false
+func (cl *ClientImpl) GetDefaultAccount() *Account {
+	addr := cl.WalletData.GetDefaultAccount().Address
```
`GetDefaultAccount` may return nil, cause panic.

laizy commented on account/client.go:
```diff
@@ -209,38 +231,52 @@ func (cl *ClientImpl) GetAccountByAddress(address common.Address) *Account {
-		return false
-	}
+// Deprecated
+func (cl *ClientImpl) ChangePassword(oldPassword []byte, newPassword []byte) bool {
```
need remove this function

laizy commented on account/client.go:
```diff
@@ -250,21 +286,23 @@ func (cl *ClientImpl) ContainsAccount(pubKey keypair.PublicKey) bool {
 	return cl.GetAccountByAddress(addr) != nil
 }
 
+// Deprecated
```
clean this function

laizy commented on account/client.go:
```diff
@@ -339,40 +378,42 @@ func (cl *ClientImpl) SaveAccount(ac *Account) error {
-			break
-		}
+		i := 0
+		accounts := map[common.Address]*Account{}
```
clean up comment-out  code

AlverLyu committed:
fix bugs

1. fix bug in account/client.go
2. fix bug of account set command
3. remove deprecated codes

AlverLyu committed:
modify client.go

AlverLyu committed:
fix bug in nodectl compiling

laizy commented:
saving wallet file should be atomic，issue #61 is created for tracking this.
